custom_components/tahoma/__init__.py

```diff
@@ -157,34 +156,32 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         _LOGGER.exception(exception)
         return False
 
-    update_interval = timedelta(
-        seconds=entry.options.get(CONF_UPDATE_INTERVAL, DEFAULT_UPDATE_INTERVAL)
-    )
-
     coordinator = OverkizDataUpdateCoordinator(
         hass,
         _LOGGER,
         name="device events",
         client=client,
         devices=devices,
         places=places,
-        update_interval=update_interval,
+        update_interval=UPDATE_INTERVAL,
         config_entry_id=entry.entry_id,
     )
 
-    _LOGGER.debug(
-        "Initialized DataUpdateCoordinator with %s interval.", str(update_interval)
-    )
+    await coordinator.async_config_entry_first_refresh()
 
-    await coordinator.async_refresh()
+    if coordinator.is_stateless:
+        _LOGGER.debug(
+            "All devices have assumed state. Update interval has been reduced to: %s",
```

Comment:
```suggestion
            "All devices have an assumed state. Update interval has been reduced to: %s",
```

```suggestion
            "All devices have are stateless. Update interval has been reduced to: %s",
```

Reply:
I took it from bdraco on the Somfy component :P